### PR TITLE
Add new recipe page

### PR DESCRIPTION
**Pull request by vsrodriguez3**
Page UI for users who wish to create their own custom recipes

### CODE REVIEW

File: src/Components/SearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
```
Comment (royemosby) [inline src/Components/SearchBar.jsx]: Where did this come from? It's not a part of NewRecipe.

File: src/Components/Title.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+.title-container h1 {
+  font-size: 4rem;
+  padding-bottom: 10px;
+}
+
+.title-container p {
+  font-size: 1.2rem;
+  padding-bottom: 50px;
+}
```
Comment (royemosby) [inline src/Components/Title.css]: The title component is being developed on the Login UI ticket which is Annie's ticket. Please remove this file and revert the Title component back to how it was. If you want, you can share these as snippets to Annie.

File: src/Components/Title.jsx [hunk 3]
```diff
@@ -1,8 +1,12 @@
-export default function Title(){
-  return(
-    <div>
-      <h1>Recipe Finder</h1>
-      <p>A tool to help you use up those leftover ingredients</p>
-    </div>
-  )
+import "./Title.css";
+
+export default function Title() {
```
Comment (royemosby) [inline src/Components/Title.jsx]: Revert to original

**Review by royemosby — CHANGES_REQUESTED** [review]
Your design is pretty faithful to the Figma prototype so good job there! There are a few items that need to be addressed before we can merge:

- NewRecipe.jsx should be in `/pages` directory, not `/Components` 
    - delete the NewRecipe.jsx file in `/pages` then move your version there.
    - Update resulting import changes
- Update the font-family for elements on your page to Inter as needed to match Figma design. It is already installed on the project so all you'll have to add is `font-family: Inter` to the appropriate selectors.
- PREP and COOK inputs should be type "number" - ignore the "mins" in the field and probably add "(in mins)" to the input labels to prevent user confusion.
- Title component should be on top of the page but don’t do any development work with it. That is a part of the Login UI ticket that Annie is working.

File: src/Components/Title.jsx (hunk 3 above)
Comment (vsrodriguez3) [reply src/Components/Title.jsx]: Hi Roy, I have made the suggested changes. Thank you

File: src/Components/SearchBar.jsx (hunk 1 above)
Comment (royemosby) [reply src/Components/SearchBar.jsx]: A SearchBar component was not a part of the work performed on the New Recipe ticket. Why was this added?

File: src/Components/Title.css (hunk 2 above)
Comment (royemosby) [reply src/Components/Title.css]: Please remove this from this PR.

File: src/Components/Title.jsx (hunk 3 above)
Comment (royemosby) [reply src/Components/Title.jsx]: Please revert the work done in this Title component work. Changes to this file should not appear in this PR.

**Review by royemosby — CHANGES_REQUESTED** [review]
The changes that I requested have not been addressed. If you are having difficulties removing the SearchBar file and the changes that you made to Title.css and Title.jsx please reach out on Slack to set up a call so I can assist.

**Review by royemosby — APPROVED** [review]
thanks for the updates

**Comment by royemosby** [comment]
I'll deal with the conflicts tomorrow 💤

**Pull request merged**